### ISSUE TITLE
Does the check for cloud_types work?

### ISSUE THREAD

**Issue by gregorwolf** [abaplint/abaplint#561]
In the [abap2xlsx](https://github.com/sapmentors/abap2xlsx) Project abaplint is configured with:

```
    "cloud_types": {
      "enabled": true
    },
```
and does not show any errors. But after the import in Steampunk (SAP CP ABAP Environment) this is one of the errors that are shown:

<img width="674" alt="Screenshot 2019-09-30 at 10 00 10" src="https://user-images.githubusercontent.com/2639535/65917427-9d6ab280-e3d7-11e9-81ce-9a87eac7d912.png">

**Comment by larshp** [comment]
it only works if the language version is also cloud, https://github.com/abaplint/abaplint/blob/master/src/rules/cloud_types.ts#L34

**Comment by larshp** [comment]
cloud_types only checks the object types, not the allowed statements, so checking for cloud you'd typically set the version to cloud

**Comment by larshp** [comment]
@gregorwolf does above help?
